CHANGELOG.md

```diff
@@ -31,6 +31,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Removed
 - No longer support Coq 8.10.
 
+## [1.2.1] - 2020-04-09
+Backport some fixes in [1.3.1] to Coq 8.10.
```

Comment:
Mention these changes are not included in v1.3.0?

Reply:
Yeah, sounds reasonable.